nomad/local/grapl-local-infra.nomad

```diff
@@ -221,7 +221,7 @@ job "grapl-local-infra" {
       driver = "docker"
 
       config {
-        image = "confluentinc/cp-kafka:6.2.0"
+        image = "confluentinc/cp-kafka:6.2.2"
```

Comment:
Why did this need to be updated in this PR?

Reply:
Ack, this wasn't supposed to be in this PR. Removing it now